apps/web/cypress/integration/outcome/outcomes.spec.ts

```diff
@@ -101,7 +98,7 @@ describe('Recommendations', () => {
     cy.contains(questions.q2.question).no().continue();
     cy.contains(questions.q3.question).selectSymptom('fever').selectSymptom('cough').continue();
     cy.contains(questions.severity.cough).get('#Mild').click().continue();
-    cy.contains(questions.hwq1.question).yes().continue();
+    cy.contains(questions.hwq1.question).no().continue();
```

Comment:
Could you explain this test change?

Reply:
Absolutely!
With the changes to their algorithm, selecting "yes" to hwq1 is now supposed to end the assessment with a recommendation to get tested (so the rest of the test code would not be executed). The hwq7/hwq8 responses _also_ trigger a recommendation to get a test, but that logic was trickier so I opted to keep that in.

Reply:
Ah so previously it didn't matter if it was yes or now, but now yes ends the survey so you've switched it to no?

Reply:
Pretty much. Now yes ends the survey so I switched it to no. There were 2 triggers in the test that would give the result.
There's definitely a case to be made for adding more tests to try out more paths.